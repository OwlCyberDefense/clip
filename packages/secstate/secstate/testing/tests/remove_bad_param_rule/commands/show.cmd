/usr/sbin/secstate show PassComp -v
