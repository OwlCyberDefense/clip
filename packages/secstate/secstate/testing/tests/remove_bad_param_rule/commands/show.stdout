PassComp:
	Title:  'Password Complexity'
	Selected:  True
	Profiles:
		[ ]emptyProfile - 'An empty profile'
		[X]None - 'None'
	Sub Elements:
		PassComp-G-2-2
